bridges/LeMondeInformatiqueBridge.php

```diff
@@ -20,12 +20,12 @@ protected function parseItem($newsItem){
 			str_replace(
 				'/grande/',
 				'/petite/',
-				$article_html->find('.article-image', 0)->find('img', 0)->src
+				$article_html->find('.article-image > img, figure > img', 0)->src
 			)
 		);
 
 		//No response header sets the encoding, explicit conversion is needed or subsequent xml_encode() will fail
-		$item['content'] = utf8_encode($this->cleanArticle($article_html->find('div.col-primary', 0)->innertext));
+		$item['content'] = utf8_encode($this->cleanArticle($article_html->find('div.col-primary, div.col-sm-9', 0)->innertext));
```

Comment:
Travis has problems with the length of this line. It's probably best to split it into multiple like so:

```suggestion
		$content = $article_html->find('div.col-primary, div.col-sm-9', 0);
		$item['content'] = utf8_encode($this->cleanArticle($content->innertext));
```